packages/editor-ui/src/plugins/i18n/locales/en.json

```diff
@@ -1390,25 +1393,29 @@
 	"settings.sourceControl.modals.refreshSshKey.message": "This will delete the current SSH key and create a new one. You will not be able to authenticate with the current key anymore.",
 	"settings.sourceControl.modals.refreshSshKey.cancel": "Cancel",
 	"settings.sourceControl.modals.refreshSshKey.confirm": "Refresh key",
+	"settings.sourceControl.loading.connecting": "Connecting",
 	"settings.sourceControl.toast.connected.title": "Git repository connected",
 	"settings.sourceControl.toast.connected.message": "Select the branch to complete the configuration",
 	"settings.sourceControl.toast.connected.error": "Error connecting to Git",
 	"settings.sourceControl.toast.disconnected.title": "Git repository disconnected",
 	"settings.sourceControl.toast.disconnected.message": "You can no longer sync your instance with the remote repository",
 	"settings.sourceControl.toast.disconnected.error": "Error disconnecting from Git",
 	"settings.sourceControl.loading.pull": "Pulling from remote",
+	"settings.sourceControl.loading.checkingForChanges": "Checking for changes",
 	"settings.sourceControl.loading.push": "Pushing to remote",
 	"settings.sourceControl.lastUpdated": "Last updated {date} at {time}",
 	"settings.sourceControl.saved.title": "Settings successfully saved",
 	"settings.sourceControl.refreshBranches.tooltip": "Reload branches list",
 	"settings.sourceControl.refreshBranches.success": "Branches successfully refreshed",
 	"settings.sourceControl.refreshBranches.error": "Error refreshing branches",
-	"settings.sourceControl.docs.url": "https://docs.n8n.io/source-control/",
-	"settings.sourceControl.docs.setup.url": "https://docs.n8n.io/source-control/setup/",
-	"settings.sourceControl.docs.using.url": "https://docs.n8n.io/source-control/using/",
+	"settings.sourceControl.docs.url": "https://docs.n8n.io/source-control-environments/",
+	"settings.sourceControl.docs.setup.url": "https://docs.n8n.io/source-control-environments/setup/",
+	"settings.sourceControl.docs.setup.ssh.url": "https://docs.n8n.io/source-control-environments/setup/#step-3-set-up-a-deploy-key",
+	"settings.sourceControl.docs.using.url": "https://docs.n8n.io/source-control-environments/using/",
+	"settings.sourceControl.docs.using.pushPull.url": "https://docs.n8n.io/source-control-environments/using/push-pull",
```

Comment:
We don't have multilingual docs - these might be better off as constants like all the others.